### PR TITLE
Fix of Device and Data Viz pages

### PR DESCRIPTION
**Pull request by bardram**
Misc fixes to

* showing location service #203 
* showing the health service #208 
* #204 & #207

### CODE REVIEW

File: lib/ui/cards/heart_rate_card.dart [hunk 1]
```diff
@@ -102,7 +102,8 @@ class HeartRateCardWidgetState extends State<HeartRateCardWidget>
           margin: const EdgeInsets.only(left: 8, right: 2),
           child: Text(
             min == null || max == null
-                ? locale.translate('cards.no_data')
+                // ? locale.translate('cards.no_data')
```
Comment (LarsRefsgaard) [inline lib/ui/cards/heart_rate_card.dart:105]: Remove dead code comments

**Comment by LarsRefsgaard** [comment]
@bardram 
Go pull `master` into your branch to fix the iOS building in the cloud, and then we can approve the PR for merging :)

**Comment by bardram** [comment]
> @bardram Go pull `master` into your branch to fix the iOS building in the cloud, and then we can approve the PR for merging :)

Done.

**Comment by LarsRefsgaard** [comment]
> > @bardram Go pull `master` into your branch to fix the iOS building in the cloud, and then we can approve the PR for merging :)
> 
> Done.
> 
> I wonder if we should add these `lock` files to `.gitignore`?

Only in libraries do you not add locks to git.
https://dart.dev/guides/libraries/private-files

**Review by LarsRefsgaard — APPROVED** [review]

**Pull request merged**